pgmax/fg/graph.py

```diff
@@ -861,14 +977,28 @@ def update(msgs: jnp.ndarray, _) -> Tuple[jnp.ndarray, None]:
                 evidence,
                 wiring.var_states_for_edges,
             )
+            ftov_msgs = vtof_msgs
```

Comment:
If this is for initialization, `ftov_msgs = jnp.zeros_like(vtof_msgs)` might be better and less confusing.